src/SYN.py

```diff
@@ -169,6 +170,11 @@ def PART_SET_TOOL(part_str, allow_fail=False):
                         print("Diamond:", DIAMOND.DIAMOND_PATH, flush=True)
                     else:
                         raise Exception("Diamond install not found!")
+            elif part_str.lower().startswith("gw1"):
+                SYN_TOOL = OPEN_TOOLS_GOWIN
+            #elif part_str.lower().startswith("gw2"):
```

Comment:
Related: whats the plan here? I had been thinking GOWIN(.py) would be the default  - but per other comment - just saw that other file

Could we do like `OPEN` appended via those colons?
`#pragma PART GW2AR-LV18QN88PC8:C:OPEN` or

Reply:
Oh, I had another branch with this, things bled to the branch that I was working on the gowin tools - I'll revert that (and the one in your previous comment)